test/mic-check.html

```diff
@@ -0,0 +1,130 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <button onclick="start()">Start!</button>
+    <div id="messages"></div>
+    🎤
+    <button onclick="recordingStart()">⏺</button>
+    <button onclick="recordingStop()">⏹</button>
+  </body>
+  <script type="module">
+    import {
+      Playthrough,
+      Speaker,
+      Microphone,
+      createPlaythroughToken,
+      createConversation,
+    } from "../dist/index.js";
+
+    import {
+      setGlobalBaseUrl,
+    } from "../dist/api.js";
+
+    // TEST PARAMETERS
+    const storyId = 2;
+    const version = -1;
+    const userToken = "eyXXX9.eyXXX.XX";
+    const startGraphReferenceId = "55f6a2ae-f526-4e50-b6ca-cfbf8cea4ffa";
+    const speechRecognitionStartEvent = {
+      service: "unified",
+      languageCode: "en-US",
+      returnRaw: false,
+      customServiceParameters: {}
+    };
+    const charismaUrl = "http://localhost:5050";
+    //
+
+    setGlobalBaseUrl(charismaUrl);
+
+    console.log();
```

Comment:
I know we're the only ones that will end up using this test file, but would be good to tidy up empty comment/console.log